### DISCUSSION TITLE
grafana should handle "snapshot post -> 413 Request Entity Too Large" better

### DISCUSSION THREAD

**Discussion by Dieterbe** [grafana/grafana#36988] [Dashboards, Panels & Visualizations]
If you post a snapshot and the request results in "413 Request Entity Too Large", you get an ugly error in grafana.
It would be better if grafana would handle this more gracefully and present a warning dialog to the user. something along the lines of
"there's too much data in this dashboard, please select a narrower timerange, or adjust which data is shown"

**Comment by yarix** [comment]
is it related \ due to #3176 ? (using proxy?)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
